File: SecondYear/semestrul4/MPP/PROIECT/CSharpProjectNetworking/Server/service/InscriereService.cs
using Model;
using Persistence;

namespace Server;

public class InscriereService : AbstractService<Tuple<Participant, Proba>, Inscriere>
{
    public InscriereService(IRepository<Tuple<Participant, Proba>, Inscriere> repository) : base(repository)
    {
    }
}

File: SecondYear/semestrul4/MPP/PROIECT/JavaProjectORM/ProjectServer/src/main/java/ro/mpp2025/javaprojectui/service/InscriereService.java
package ro.mpp2025.javaprojectui.service;


import ro.mpp2025.javaprojectui.*;

public class InscriereService extends AbstractService<Tuplu<Participant, Proba>, Inscriere>{
    public InscriereService(Repository<Tuplu<Participant, Proba>, Inscriere> repository) {
        super(repository);
    }
}
